### PR TITLE
gh-107017: Analolgy to Pascal and C replaced.

### PR DESCRIPTION
**Pull request by TommyUnreal**
Knowledge of arithmetic is more accessible than knowledge of other programming languages. An analogy is, therefore, more accessible.

* Issue: gh-107017

----
:books: Documentation preview :books:: https://cpython-previews--107025.org.readthedocs.build/

### CODE REVIEW

**Review by evildmp — APPROVED** [review]

**Review by serhiy-storchaka — COMMENTED** [review]
In mathematics you use `·`, `×`, `:` and `÷`.

`*` for multiplication is only used in programming.

**Comment by evildmp** [comment]
> In mathematics you use `·`, `×`, `:` and `÷`.
> 
> `*` for multiplication is only used in programming.

It's standard and familiar to most people who use computers - anyone who has used a spreadsheet application, for example. I think that it's also made pretty clear in the examples, which in a tutorial is much more effective than complicating a brief explanation.

File: Doc/tutorial/introduction.rst [hunk 1]
```diff
@@ -52,8 +52,8 @@ Numbers
 
 The interpreter acts as a simple calculator: you can type an expression at it
 and it will write the value.  Expression syntax is straightforward: the
-operators ``+``, ``-``, ``*`` and ``/`` work just like in most other languages
-(for example, Pascal or C); parentheses (``()``) can be used for grouping.
+operators ``+``, ``-``, ``*`` and ``/`` work similarly to how you perform
+arithmetic operations in mathematics; parentheses (``()``) can be used for grouping.
```
Comment (hauntsaninja) [inline Doc/tutorial/introduction.rst]:
```suggestion
operators ``+``, ``-``, ``*`` and ``/`` can be used to perform
arithmetic; parentheses (``()``) can be used for grouping.
```
more concise, slightly more informal, no need to use "similarly" to plant doubts in beginners' minds

**Review by hauntsaninja — COMMENTED** [review]
I think it's true that `*` and `/` have so completely won out that a) newcomers from C or Pascal don't need to be told they can be used for arithmetic, b) most people who've used computers would not find this surprising, c) `/` doesn't really work like C in Python 3

My wording suggestion therefore minimises length

**Comment by TommyUnreal** [comment]
> I think it's true that `*` and `/` have so completely won out that a) newcomers from C or Pascal don't need to be told they can be used for arithmetic, b) most people who've used computers would not find this surprising, c) `/` doesn't really work like C in Python 3
> 
> My wording suggestion therefore minimises length

Hi! I appreciate your suggestion. I like the wording better as it holds the same information but in a more condensed format.

**Review by hugovk — APPROVED** [review]
Thank you!

**Pull request merged**